core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -984,51 +986,56 @@ public GroupByIdBlock getResult()
             implements Work<T>
```

Comment:
Since you removed most of the fields (I don't think you even use `this.page`), I don't think there is a need for `abstract LowCardinalityDictionaryWork<T>` class anymore. Let's just leave methods

Reply:
Class removed in favor of methods.